caput/util/profiler.py

```diff
@@ -318,30 +325,12 @@ def start(self):
         self._monitor_thread = threading.Thread(target=self.monitor)
         self._monitor_thread.start()
 
-    def stop(self):
-        """Stop profiler. Dump results to csv file and/or log and/or set results on self.usage.
+    def stop(self) -> None:
+        """Stop profiler.
```

Comment:
```suggestion
        """Stop profiling.
```
For consistency with `start`.